code/modules/mob/living/simple_animal/friendly/drone/extra_drone_types.dm

```diff
@@ -129,7 +129,7 @@
 
 /mob/living/simple_animal/drone/derelict/Initialize()
 	. = ..()
-	AddComponent(/datum/component/stationstuck, TRUE, "Your emergency station return device activates, sending you back to KS13!", "01000110 01010101 01000011 01001011 00100000 01011001 01001111 01010101<br>WARNING: Dereliction of KS13 detected. Self-destruct activated.")
+	AddComponent(/datum/component/stationstuck, PUNISHMENT_GIB, "01000110 01010101 01000011 01001011 00100000 01011001 01001111 01010101<br>WARNING: Dereliction of KS13 detected. Self-destruct activated.")
```

Comment:
Why'd you change this to a gibbing punishment? Now people can grief derelict drones using hand teles.

Reply:
It already was a gib punishment, it was just broken, so would use a teleport message for the gib.